### ISSUE TITLE
Filter output for "pw3nd" when spray across big scopes

### ISSUE THREAD

**Issue by evildrummer** [Pennyw0rth/NetExec#85]
Hey,

it would be cool to get a filter like "--only-admin" when spaying vaild credentials over many hosts. In my PenTests i spray newly gathered creds a gsinst all clients to check for admin access.

Yes I could grep for it but with the filter the colors would stay and it's much nicer for Screenshots :)

**Comment by Marshall-Hallenbeck** [comment]
This was brought up before doing something like `--only-success` but several people didn't like it. I however, do like it and think we should bring it back. @NeffIsBack can you scrounge up the code you wrote for it?

**Comment by NeffIsBack** [comment]
Yes that would be possible. What came to my mind was that we just could add a check for the specific log message instead of the previous system where we disabled all info/error messages. Thoughts?

**Comment by Marshall-Hallenbeck** [comment]
I think allowing explicitly disabling the login failures is preferrable, at least in the use cases I would want to.
